content/applications/finance/accounting/get_started/journals.rst

```diff
@@ -0,0 +1,272 @@
+outstanding payments account's balance in the general ledger shows the balance of registered
+outgoing payments that have not yet been reconciled.
+
+Bank and cash journal configuration
+***********************************
+
+To configure :doc:`payments <../payments>` to create journal entries, set outstanding accounts for
+the journal's payment methods. This can be done for any journal with the :ref:`type
+<chart-of-account/type>` :guilabel:`Bank` or :guilabel:`Cash`.
+
+To configure the outstanding accounts for a journal's payment methods, first go to
+:menuselection:`Accounting --> Configuration --> Journals` and select a bank or cash journal. In the
+:guilabel:`Incoming Payments` and :guilabel:`Outgoing Payments` tabs, set :guilabel:`Outstanding
+Receipts accounts` and :guilabel:`Outstanding Payments accounts` for each payment method that you
+want to create journal entries.
+
+.. note::
+   - If the main bank account of the journal is added as an outstanding receipts account or
+     outstanding payments account, when a payment is registered, the invoice or bill's status is
+     directly set to :guilabel:`Paid`.
+   - If the outstanding receipts or outstanding payments account for a payment method is left blank,
+     registering a payment with that payment method will not create any journal entry.
+
+.. _accounting/journals/bank:
+
+Bank
+~~~~
+
+Bank journals are used to record journal entries related to :doc:`bank transactions
+<../bank/transactions>` and incoming and outgoing :doc:`payments <../payments>`. The following
+fields are specific to bank journals:
+
+- :guilabel:`Bank Account`: This :guilabel:`Bank and Cash` type account is the default account for
+  this bank journal.
+- :guilabel:`Account Number`: The bank account's number is used when registering payments and is
+  required for generating outgoing payment files, such as :doc:`SEPA <../payments/pay_sepa>` or
+  :ref:`NACHA <l10n_us/nacha>`. To edit the bank account details, click on the external link arrow
+  next to the :guilabel:`Account Number` and update the account information accordingly.
+- :guilabel:`Bank`: The bank name is used when registering payments and is required for generating
+  outgoing payment files, such as :doc:`SEPA <../payments/pay_sepa>` or :ref:`NACHA
+  <l10n_us/nacha>`. To edit the bank account details, click on the external link arrow next to the
+  :guilabel:`Bank` name and update the bank information accordingly.
+- :guilabel:`Bank Feeds`: Define the method of creating bank :doc:`transactions
+  <../bank/transactions>`, whether manual or via :doc:`Online Synchronization
+  <../bank/bank_synchronization>`.
+- :guilabel:`Split Transactions`: Split collective payments for CODA files.
+
+Multiple payment methods are available for bank journals, as are configurations for generating
+outgoing payment files, such as :doc:`SEPA <../payments/pay_sepa>` or :ref:`NACHA <l10n_us/nacha>`.
```

Comment:
I think giving the example of SEPA and NACHA payment files once is sufficient; maybe keep that one here at the end?